internal/indexnode/indexnode.go

```diff
@@ -197,19 +220,20 @@ func (i *IndexNode) DropIndex(ctx context.Context, request *indexpb.DropIndexReq
 	indexBuildIDs := i.sched.IndexBuildQueue.tryToRemoveUselessIndexBuildTask(request.IndexID)
 	log.Debug("IndexNode", zap.Any("The index of the IndexBuildIDs to be deleted", indexBuildIDs))
 	for _, indexBuildID := range indexBuildIDs {
-		nty := &indexpb.NotifyBuildIndexRequest{
-			Status: &commonpb.Status{
-				ErrorCode: commonpb.ErrorCode_Success,
-			},
-			IndexBuildID:   indexBuildID,
-			NodeID:         Params.NodeID,
-			IndexFilePaths: []string{},
+		metaPath := "/indexes/" + strconv.FormatInt(indexBuildID, 10)
```

Comment:
etcd kv root path should be xxx/indexes/

Reply:
There is no need to slove it temporarily, I will fix it soon.